src/lib/realizeCustomElements.ts

```diff
@@ -234,6 +234,10 @@ function resolveOptions(registry: ReadOnlyRegistry, registryProvider: RegistryPr
 	return options;
 }
 
+function getTransitionOptionFromProjector(element: Element): boolean {
+	return element.getAttribute('data-css-transitions') === 'true';
```

Comment:
should support, `<app-projector data-css-transitions>`, `<app-projector data-css-transitions="true">` and `<app-projector data-css-transitions="false">` now